### DISCUSSION TITLE
CORS config issue.

### DISCUSSION THREAD

**Discussion by spmishra121** [DependencyTrack/dependency-track#2261] [Q&A] [answered]
I am getting below error while running DT. Please help me to resolve this...

![image](https://user-images.githubusercontent.com/6615148/206691944-bc88de92-d12f-4711-96e3-79351d7e21a8.png)

Docker-compose file:

![image](https://user-images.githubusercontent.com/6615148/206692179-0e509021-675d-43a3-8dbd-53f252bcb16e.png)

**Comment by nscuro** [comment, marked as answer]
Your API server is responding with `HTTP 502` (Bad Gateway). That error response most likely does not set any CORS headers.

Based on the way you're trying to access DT, you have some kind of reverse proxy in place. Review the reverse proxy config and ensure it correctly routes through to the API server.
